packages/web/utils/dework.ts

```diff
@@ -61,9 +77,23 @@ export const processDeworkData = (data: DeworkData) => {
     });
     // get usdc rewards
     element?.rewards?.forEach(({ token, amount }) => {
-      if (usdcTokens.includes(token.address || 'empty')) return;
-      if (amount) {
-        totalEarnedInUSDC += +amount;
+      if (
+        usdcTokens6Decimals.includes(token.address?.toLowerCase() || 'empty') &&
+        amount
+      ) {
+        totalEarnedInUSDC += +amount / 10 ** 6;
+      }
+      if (
+        usdTokens18Decimals.includes(token.address?.toLowerCase() || 'empty') &&
+        amount
+      ) {
+        totalEarnedInUSDC += +amount / 10 ** 18;
```

Comment:
This actually works? It seems like `amount` could overflow an `int` and then get divided by 10¹⁸.

Do we need to be using `ethers.toWei()`?